boolean_algebra/and_gate.py

```diff
@@ -28,7 +28,11 @@ def and_gate(input_1: int, input_2: int) -> int:
     0
     >>> and_gate(1, 1)
     1
+    >>> and_gate(2, 1)  # Added test case with invalid input
+    "Please enter valid input values: 0 or 1"
     """
+    if input_1 not in (0, 1) or input_2 not in (0, 1):
+        return "Please enter valid input values: 0 or 1"
```

Comment:
This should raise an error instead

Reply:
There is no errors raises i got a successful output while executing it